mitosheet/src/mito/Mito.tsx

```diff
@@ -411,7 +411,7 @@ export const Mito = (props: MitoProps): JSX.Element => {
         }
 
         // Close the cell editor if it is open
-        if (editorState !== undefined) {
+        if (editorState !== undefined && !editorState.formula.includes('VLOOKUP')) {
             setEditorState(undefined)
```

Comment:
I actually think we should add the cross-sheet reference even if the user doesn't have VLOOKUP written. Namely, if they can't add cross cell references they will be very confused why. Instead, if we let them add it - and then throw that helpful error you have on the backend, they will learn why that doesn't work!